packages/foam-core/tsconfig.json

```diff
@@ -6,7 +6,7 @@
     "esModuleInterop": true,
     "importHelpers": true,
     "downlevelIteration": true,
-    "target": "es2019",
+    // "target": "es2019",
```

Comment:
I saw the discussion of this in the discord. Mind summarising it in a brief one line comment following this?

Reply:
oh yeah, good point, I missed it!